### PR TITLE
enable profiles

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/145
image
image
image

### CODE REVIEW

File: Services/Interfaces/web_interface/controllers/configuration.py [hunk 1]
```diff
@@ -26,108 +28,168 @@
...
+                                 in_backtesting=backtesting_api.is_backtesting_enabled(display_config),
+
+                                 config_tentacles_by_group=models.get_tentacles_activation_desc_by_group(media_url,
+                                                                                                         missing_tentacles)
+                                 )
+
+
+@web_interface.server_instance.route('/profiles_management/<action>', methods=["POST", "GET"])
+@login.login_required_when_activated
+def profiles_management(action):
```
Comment (Herklos) [inline Services/Interfaces/web_interface/controllers/configuration.py:78]: :+1:

File: Services/Interfaces/web_interface/controllers/configuration.py [hunk 2]
```diff
@@ -26,108 +28,168 @@
 import octobot_services.interfaces.util as interfaces_util
 
 
-@web_interface.server_instance.route('/config', methods=['GET', 'POST'])
+@web_interface.server_instance.route('/profile')
+@login.login_required_when_activated
```
Comment (Herklos) [inline Services/Interfaces/web_interface/controllers/configuration.py:32]: :+1:

**Review by Herklos — COMMENTED** [review]
Very nice job !

**Comment by Herklos** [comment]
Great feature !

**Review by Herklos — APPROVED** [review]
I can't wait to use this feature :100:

**Comment by GuillaumeDSM** [comment]
contained in https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/343

**Pull request closed without merging**